src/main/java/com/saucelabs/bamboo/sod/plan/ViewSODCondition.java

```diff
@@ -12,6 +19,16 @@
  * @author Ross Rowe
  */
 public class ViewSODCondition implements Condition {
+    protected PlanManager planManager;
```

Comment:
setters are for spring. Getter is actually never used.

I generally don't like private but I guess it makes sense here.
On Apr 18, 2016 6:01 PM, "Moiz Virani" <notifications@github.com> wrote:

> In src/main/java/com/saucelabs/bamboo/sod/plan/ViewSODCondition.java
> <https://github.com/saucelabs/bamboo_sauce/pull/39#discussion_r60160007>:
>
> > @@ -12,6 +19,16 @@
> >   * @author Ross Rowe
> >   */
> >  public class ViewSODCondition implements Condition {
> > +    protected PlanManager planManager;
>
> why not private when there are getters and setters?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/saucelabs/bamboo_sauce/pull/39/files/67b7543aeafefa5a93bf12cf0cf28e9326e5777b#r60160007>
>